python_modules/dagster-test/dagster_test/fixtures/docker_compose.py

```diff
@@ -12,15 +12,12 @@
 
 @contextmanager
 def docker_compose_cm(
-    docker_compose_yml,
-    network_name=None,
-    docker_context=None,
-    service=None,
+    docker_compose_yml, network_name=None, docker_context=None, service=None, env_file=None
```

Comment:
Nice - so we no longer need to monkeypatch env in the tests?

Reply:
Because the docker compose test fixtures are kind of...complicated...would you mind adding a unit test for this? Similar to the existing ones but passing an env file?